cognite/neat/_session/_physical.py

```diff
@@ -20,6 +21,12 @@ def __init__(self, store: NeatStore, client: NeatClient) -> None:
         self.read = ReadPhysicalDataModel(self._store, self._client)
         self.write = WritePhysicalDataModel(self._store)
 
+    def __call__(self) -> RequestSchema | None:
```

Comment:
This is not used. 

Note that we also do not have a html representation of the RequestSchema so this will not look nice.